RELEASES.md

```diff
@@ -19,9 +19,13 @@ Release 0.8.5 on 2017-06-??
   by Anderson Frailey]
 - Add policy reform file for the Renacci reform
   [[#1376](https://github.com/open-source-economics/Tax-Calculator/pull/1376)
-  and
-  [#1383](https://github.com/open-source-economics/Tax-Calculator/pull/1383)
-  by Hank Doupe]
+  ,[#1383](https://github.com/open-source-economics/Tax-Calculator/pull/1383)
+  , and
+  [#1385](https://github.com/open-source-economics/Tax-Calculator/pull/1385)
+  by Hank Doupe
+- Add separate ceiling for each itemized deduction parameter
+  [#1385](https://github.com/open-source-economics/Tax-Calculator/pull/1385)
+  by Hank Doupe
```

Comment:
Check your double brackets here. You can view how the file is rendering 
![image](https://user-images.githubusercontent.com/8301092/26936818-aa04cd0c-4c3d-11e7-8be6-1515b29021d5.png)